components/Accordion/__snapshots__/Accordion.unit.test.jsx.snap

```diff
@@ -95,111 +95,31 @@ exports[`Accordion component Should match the snapshot 1`] = `
 }
 
 <ul
-  className="c0"
+  class="c0"
 >
   <li
-    className="c1 accordion-item"
+    class="c1 accordion-item"
     color="#191919"
   >
     <button
       aria-controls="Institucional-0-content"
-      className="c2"
+      class="c2"
       id="Institucional-0-header"
-      onClick={[Function]}
```

Comment:
Don't snaps show anymore theme and events? I don't know if this is a problem. What do you think?